packages/ketcher-react/src/script/ui/views/components/Dialog/Dialog.tsx

```diff
@@ -81,7 +81,7 @@ const Dialog: FC<Props> = (props) => {
   }, [])
 
   const isButtonOk = (button) => {
-    return button === 'OK' || button === 'Save'
+    return button === 'OK' || button === 'Save' || button === 'Apply'
```

Comment:
'Apply' option should be deleted

Reply:
We set Apply button instead of OK button. What should we do instead of this decision?

Reply:
done